taxi/models.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+# Create your models here.
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        verbose_name = "manufacturer"
+        verbose_name_plural = "manufacturers"
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return self.username
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    driver = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars")
+
+    class Meta:
+        verbose_name = "car"
+        verbose_name_plural = "cars"
```

Comment:
There is no need for defining `verbose_name` and `verbose_name_plural` for these classes, you could rely on the default implementation.